### PR TITLE
Doc reorg to include Tidy3D Python tutorials in the learning center

### PR DESCRIPTION
**Pull request by e-g-melo**
We will include those introductory Python notebooks (from basics to adjoint) on learning center. I've proposed this reorganization to have a more uniform categorization on both Tidy3D Python and Tidy3D GUI tutorials.

### CODE REVIEW

**Comment by tomflexcompute** [comment]
Thanks Emerson. The reorganization looks good to me. I just noticed that due to the URL change, the links in the inversedesign.rst are not correct anymore. Could you also correct them to use the latest links? The fdtd101 links seem fine but feel free to double check as well. Thanks.

**Comment by e-g-melo** [comment]
Hi Tom! I've updated the inverse design URLs. The FDTD 101 are already updated.

**Review by tomflexcompute — APPROVED** [review]
Thanks!

**Review by tylerflex — CHANGES_REQUESTED** [review]
Looks good to me, the only thing is that I think in RST the underline characters must be the same lines as the titles, so for example
```
Mediums
~~~~~~~~~~~~~~~~~~~~~~~~
```
should be
```
Mediums
~~~~~~~
```
could you change these please? thanks

EDIT: turns out they just need to be **at least as long** but as a convention let's make them the same length so we know they will always work

https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

**Comment by e-g-melo** [comment]
Hi @tylerflex! I've corrected the file.

File: docs/source/examples.rst [hunk 1]
```diff
@@ -19,76 +19,97 @@ Tidy3D Basics
...
+~~~~~~~~~~~
 
 .. toctree::
     :maxdepth: 1
 
     notebooks/ModeSolver
     notebooks/WaveguidePluginDemonstration
-    notebooks/FieldProjections
+
+Parameter Sweep
```
Comment (tylerflex) [inline docs/source/examples.rst:98]: do we want to include PSO stuff here?

File: docs/source/examples.rst [hunk 2]
```diff
@@ -19,76 +19,97 @@ Tidy3D Basics
...
-    notebooks/WebAPI
-    notebooks/ParameterScan
+    notebooks/ModalSourcesMonitors
+    notebooks/ModesBentAngled
+    notebooks/TFSF
+    notebooks/CustomFieldSource
 
-Visualization of Simulation and Monitor Data
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Monitors and Data Visualization
```
Comment (tylerflex) [inline docs/source/examples.rst:78]: maybe "Data Visualization and Postprocessing"?

File: docs/source/inversedesign.rst [hunk 3]
```diff
@@ -4,8 +4,10 @@ Inverse design in photonics
...
 
-`Lecture 1: Introduction to Inverse Design In Photonics <https://www.flexcompute.com/fdtd101/Lecture-11-Introduction-to-Inverse-Design-In-Photonics/>`_
+`Lecture 1: Introduction to Inverse Design In Photonics <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Lecture-1-Introduction-to-Inverse-Design-In-Photonics/>`_
 
-`Lecture 2: Adjoint Method <https://www.flexcompute.com/fdtd101/Lecture-12-Inverse-Design-in-Photonics-Lecture-2-Adjoint-Method/>`_
+`Lecture 2: Adjoint Method <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Lecture-2-Inverse-Design-in-Photonics-Lecture-2-Adjoint-Method/>`_
+
+`Lecture 3: Adjoint Optimization <https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Inverse-Design-in-Photonics-Lecture-3-Adjoint-Optimization/>`_
```
Comment (tylerflex) [inline docs/source/inversedesign.rst:11]: lecture 4 is out now

https://www.flexcompute.com/tidy3d/learning-center/inverse-design/Inverse-Design-in-Photonics-Lecture-4-Fabrication-Constraints/

**Review by tylerflex — APPROVED** [review]
looks fine to merge after these changes (if you agree!)

File: docs/source/examples.rst (hunk 2 above)
Comment (e-g-melo) [reply docs/source/examples.rst:78]: OK.. nice!

File: docs/source/inversedesign.rst (hunk 3 above)
Comment (e-g-melo) [reply docs/source/inversedesign.rst:11]: Included!

**Pull request merged**